### DISCUSSION TITLE
Push code to GIthub on every compilation

### DISCUSSION THREAD

**Discussion by ssk1216** [atom/atom#22570] [General]
Is there a way to push the code from ATOM to Github repo whenever a compilation is done?

What is that we could do to achieve this?

**Comment by UziTech** [comment]
The best way would be to add this as a compilation step.

In node that would look something like:

```js
// package.json
"scripts": {
  "build": "[some build step] && git add . && git commit -m build && git push"
}
```

Then you would just run `npm run build` to push the updated files automatically after compilation.
